frontend/src/components/dor/QualifiedDashboard/QualifiedApplicationsTable.tsx

```
@@ -522,29 +325,38 @@ export default function QualifiedApplicationsTable({
-                    interviews: rowData.interviews,
-                  })
-                }
-                responseId={rowData.responseId}
-                role={rowData.role}
-              />
-            );
+            const interviewers = getValue();
+            const interviews = row.original.interviews;
+            const role = row.original.role;
+            
+            const complete = useCallback(
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)